packages/curvenote/src/sync/clone.ts

```diff
@@ -39,26 +38,29 @@ export async function interactiveCloneQuestions(
     }
   }
   let path: string;
-  const defaultPath = join('content', project.data.name);
+  const defaultPath = '.';
```

Comment:
So in curvenote's launchpad-server, path comes into this function as `undefined` which means the default path is applied, this change also modifies the behaviour of the client library based usage (e.g. `sync.clone`) so that with no change to the calling code, we also get the same "flat project" behaviour as with `curvenote init`. That seems good!?